aep/general/0144/aep.md.j2

```diff
@@ -28,7 +40,7 @@ interface Book {
   - If an array data can not be bounded (in other words, if there is a chance
     that the array will be too large to be reasonably returned in a single
     request), the API **should** use a sub-resource instead.
-- Array fields **must not** represent the body of another resource inline.
+- Array fields **should not** represent the body of another resource inline.
```

Comment:
Do we need this guidance at all?  It seems duplicative of aep.dev/association.

Reply:
SGTM - removed.